elements/pfe-band/demo/index.html

```diff
@@ -113,9 +113,11 @@ <h1>Lightest band, header region</h1>
         <p>Consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. </p>
         <p>At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.</p>
       </article>
-      <pfe-cta pfe-priority="primary" slot="pfe-band--footer">
+<footer slot="pfe-band--footer">
```

Comment:
Just curious, why update this mark-up in this PR?

Reply:
I believe it was because it was stretching the CTA across the bottom of the band.

Reply:
@castastrophe  I put two new bands at the bottom of the [band demo page](https://deploy-preview-895--happy-galileo-ea79c4.netlify.app/elements/pfe-band/demo/)

One has the CTA assigned to the card footer slot, the other has the CTA in the band body area which is using grid. But both CTA's look normal now (not stretched). Can you tell me if this is expected / ok?